src/mjolnir/graphbuilder.cc

```diff
@@ -1610,7 +1611,7 @@ void GraphBuilder::AddPronunciationsWithLang(std::vector<std::string>& pronuncia
 
   auto get_pronunciations = [](const std::vector<std::string>& pronunciation_tokens,
                                const std::vector<baldr::Language>& pronunciation_langs,
-                               const std::map<size_t, size_t> indexMap, const size_t key,
+                               const std::map<size_t, size_t>& indexMap, const size_t key,
```

Comment:
I always wondered if it is so obvious, why compiler is unable to do it on its own... It's really great to finally have this linter in the codebase.